sound/soc/intel/boards/cml_rt1011_rt5682.c

```diff
@@ -496,7 +511,7 @@ static struct snd_soc_card snd_soc_card_cml = {
 	.dai_link = cml_rt1011_rt5682_dailink,
 	.num_links = ARRAY_SIZE(cml_rt1011_rt5682_dailink),
 	.codec_conf = rt1011_conf,
-	.num_configs = ARRAY_SIZE(rt1011_conf),
+	.num_configs = 2,	/* initialize with 2 channels */
```

Comment:
It should be fine to set ``.num_configs = ARRAY_SIZE(rt1011_conf),``. We can always set it to 4 even in 2 speakers case.